tests/tt_eager/python_api_testing/unit_testing/misc/test_numcores_to_corerangeset_subcoregrids.py

```diff
@@ -0,0 +1,91 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+
```

Comment:
Nit. Can you rename the file to match the function name. ex num_cores instead of numcores

Reply:
Done